fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -534,6 +534,9 @@ public class GlobalStateMgr {
 
     private JwkMgr jwkMgr;
 
+    // The executor to handle http requests asynchronously
+    private final ThreadPoolExecutor httpAsyncExecutor;
```

Comment:
would it be good to put into HttpServer module?

Reply:
fixed